src/components/NodeEditable.tsx

```diff
@@ -132,14 +138,14 @@ class NodeEditable extends Component<Props> {
   }
 
   setSelection = (isCollapsed: boolean) => {
-    window.requestAnimationFrame(() => setTimeout(() => {
+    this.pendingTimeout = setAfterDOMUpdate(() => {
```

Comment:
async code is hard. If I call `setSelection()` multiple times and then unmount, only the last call will have its pendingTimeout canceled. You either need to keep an array of handles and cancel them all, or cancel the previous one before creating the new one.

Reply:
I don't see why we need to keep an array around. Shouldn't any call to `setSelection` just clear a previous timeout?

Reply:
seems reasonable to me.